Update GNOME core apps and remove Anjuta

GerryT11 opened the pull request:
Commit 1:
I had a look at the official GNOME app list (here: https://apps.gnome.org/) and I noticed that many applications in the list had the tag #gnome, but they are either not official GNOME apps anymore (e.g. because they are outdated and/or never started to adopt GTK4), for example Rythmbox or Evolution, or they have never been official GNOME apps. Therefore, I compared the list and the website apps.gnome.org and I removed the tags where they were not matching.

I actually was surprised with Shotwell, Gnote, Getting Things GNOME, but they are not listed in apps.gnome.org

There are a number of apps in apps.gnome.org which are not yet in the list, but so far I haven't yet added them.


Commit 2: I changed Anjuta from GTK2 to GTK3, because it was already ported, before the project was abandoned.

Commit 3: Loupe is now called "Image Viewer" as per https://apps.gnome.org/

GerryT11 committed:
Anjuta was already GTK3, before being retired

Anjuta was already GTK3, before being retired
https://gitlab.gnome.org/Archive/anjuta/-/blob/master/configure.ac?ref_type=heads

GerryT11 committed:
Removed outdated (or never accepted) #gnome tags

I compared the #gnome tags with the list of official Gnome applications. The result is that I removed many #gnome tags, because the apps or tags are outdated or the applications were never official Gnome applications.

For some I wondered myself, e.g. Gnote or Shotwell, but they are not listed on the official Gnome website.

source:
https://apps.gnome.org/

GerryT11 committed:
Loupe is now called "Image Viewer"

Loupe is now called "Image Viewer" as per https://apps.gnome.org/

loicreynier commented on README.md:
```diff
@@ -699,13 +699,13 @@ Clients for commercial social platforms that had their API access cut off in a w
 #### Featureful IDEs
 
-- [Anjuta](https://gitlab.gnome.org/Archive/anjuta) - (archived) IDE with a GUI designer for the GNOME desktop `#c` `#gtk2` `#gnome`.
+- [Anjuta](https://gitlab.gnome.org/Archive/anjuta) - (archived) IDE with a GUI designer for the GNOME desktop `#c` `#gtk3`.
```
```suggestion
```
I suggest removing Anjuta from the list, as it has been archived.

loicreynier commented:
Many applications have indeed been removed from the "core GNOME apps" recently—thanks for the update!"

GerryT11 committed:
Remove Anjuta

Yes, I agree to remove it. It is 'officially' an archived project. The last commit was two years ago. I don't know if the GTK3 ever was in production quality. And there had been a request to remove it from Ubuntu: "(From Debian) RoM; unmaintained IDE, use gnome-builder instead; Debian bug #1018726" 
https://launchpad.net/ubuntu/noble/amd64/anjuta

Co-authored-by: Loïc Reynier <88983487+loicreynier@users.noreply.github.com>
